pysph/sph/acceleration_eval_cython.mako

```diff
@@ -37,17 +40,21 @@ ${indent(all_eqs.get_py_initialize_code(), 0)}
 #######################################################################
```

Comment:
You could also add a line for the `py_initialize`?  Although this will need to be done in the helper and not in the mako.

Reply:
I've added the `with profile_ctx(...)` in the `equation.py` file.